### PR TITLE
add relative paths to import.ck

### PR DESCRIPTION
**Pull request by Arseniusz**
Hi! I add me.dir() to paths in import.ck

### CODE REVIEW

File: import.ck [hunk 1]
```diff
@@ -21,456 +21,456 @@
...
-Machine.add("lick/fn/QuaternaryFunction.ck");
-Machine.add("lick/fn/QuaternaryPredicate.ck");
-Machine.add("lick/fn/Supplier.ck");
-Machine.add("lick/fn/TernaryFunction.ck");
-Machine.add("lick/fn/TernaryPredicate.ck");
-Machine.add("lick/fn/TernaryProcedure.ck");
-Machine.add("lick/fn/UnaryFunction.ck");
-Machine.add("lick/fn/UnaryPredicate.ck");
-Machine.add("lick/fn/UnaryProcedure.ck");
+Machine.add(me.dir() + "lick/fn/BinaryFunction.ck");
```
Comment (heuermh) [inline import.ck]: What is the use case for this?  When I run stuff with LiCK I usually
```bash
$ chuck --loop &
$ chuck + import.ck
$ chuck + examples/foo.ck
```

File: import.ck [hunk 2]
```diff
@@ -21,456 +21,456 @@
...
+Machine.add(me.dir() + "lick/ladspa/swh/FreqShift.ck");
+Machine.add(me.dir() + "lick/ladspa/swh/PitchScale.ck");
+Machine.add(me.dir() + "lick/ladspa/swh/TapeDelay.ck");
 
 // RubberBand plugins, package ladspa.rubberband
-Machine.add("lick/ladspa/rubberband/RubberBand.ck");
+Machine.add(me.dir() + "lick/ladspa/rubberband/RubberBand.ck");
 
-<<<"LiCK imported.">>>;
+<<<"lick imported.">>>;
```
Comment (heuermh) [inline import.ck]: Sorry, can't do this, it is "LiCK" after all  :)

**Comment by heuermh** [comment]
I didn't mean to shoot down your suggestion -- I am curious how this helps.

**Comment by Arseniusz** [comment]
I don't remember. It was a problem with importing lib few months ago, but now I can't realize why exactly.

**Comment by heuermh** [comment]
Fixed in #9, sorry I wasn't able to merge this pull request at the time.

**Pull request closed without merging**